client/src/pages/pot-instance/components/LandingPage/LandingPage.tsx

```diff
@@ -28,55 +83,44 @@ const LandingPage: FC<Props> = ({
       <div id={styles.selectPot}>
         <div className={styles.selectItem}>
           Select Pot Type
-          <div className={styles.selectType}>
-            <button
-              type="button"
-              onClick={() => {
-                updatePotType(1);
-              }}
-              className={styles.flavors}
-            >
+          <ToggleButtonGroup
+            exclusive
+            value={potType}
+            onChange={handleChange}
+            className={styles.selectType}
+          >
+            <ToggleButton value="1" className={styles.flavors}>
               One Flavor
-            </button>
-            <button
-              type="button"
-              onClick={() => {
-                updatePotType(2);
-              }}
-              className={styles.flavors}
-            >
+            </ToggleButton>
+            <ToggleButton value="2" className={styles.flavors}>
               Two Flavors
-            </button>
-            <button
-              type="button"
-              onClick={() => {
-                updatePotType(4);
-              }}
-              className={styles.flavors}
-            >
-              Four Flavors
-            </button>
-          </div>
+            </ToggleButton>
+          </ToggleButtonGroup>
         </div>
         <div className={styles.selectBroth}>
-          Select Broth
-          <div className={styles.selectBrothType}>
-            <button type="button" className={styles.soupbases}>
-              Sukiyaki Soupbase
-            </button>
-            <button type="button" className={styles.soupbases}>
+          Select Soupbase
+          <ToggleButtonGroup
+            color="secondary"
+            // exclusive is here for the onchange to return a single value rather than an array.
+            // the single value is then processed to show the behavior we want
+            exclusive
+            value={soupbase}
+            onChange={handleChangeSoup}
+            className={styles.selectBrothType}
+          >
+            <ToggleButton value="spicy" className={styles.soupbases}>
               Spicy Soupbase
-            </button>
-            <button type="button" className={styles.soupbases}>
+            </ToggleButton>
+            <ToggleButton value="bone" className={styles.soupbases}>
               Ox Bone Soupbase
-            </button>
-            <button type="button" className={styles.soupbases}>
+            </ToggleButton>
+            <ToggleButton value="tomato" className={styles.soupbases}>
               Tomato Soupbase
-            </button>
-          </div>
+            </ToggleButton>
```

Comment:
Going back to the magic number comment above, if you can store these as a list, and map them it would make it easier to control options in the future.

Reply:
Thank you! Made it MUCH better!!